client/src/client.rs

```diff
@@ -1017,6 +1017,21 @@ pub trait RpcApi: Sized {
         self.call("uptime", &[])
     }
 
+    /// Submit a raw block
+    fn submit_block(&self, block: &[u8]) -> Result<()> {
+        let block_hex: String = block.to_hex();
+        self.submit_block_hex(&block_hex)
+    }
+
+    /// Submit a block as a hex string
+    fn submit_block_hex(&self, block_hex: &String) -> Result<()> {
```

Comment:
Should be `&str`. Everything looks good other than that!

Reply:
Addressed in https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/151/commits/406e8c30fe9a4c7e4b9ab69eda2d1bb78b280be1